docs/markdoc-component-documentation.md

```diff
@@ -0,0 +1,263 @@
+
+## Complex Header Where You Definitely Want to Use an Anchor Link {% #complex-header %} 
+
+```
+## Complex Header Where You Definitely Want to Use an 
+Anchor Link {\% #complex-header %} 
+```
+
+# Vanilla Markdown Nodes
+
+**Bold**
+
+_Italic_
+
+[Links](/docs/nodes)
+
+Vanilla markdown images work, but we should really be using the image tag because it optimizes the images, makes them more accessible, and handles resizing them for us.
+![Images](/images/concepts/assets/asset-activity-observation.png)
+
+Lists
+- Item 1
+- Item 1
+- Item 1
+
+> Quotes
+
+`Inline code`
+
+```
+Code fences
+```
+
+Markdoc doesn't have syntax highlighting out of the box. We're going to need to add that in. When it get's added in, this python code should be beautifully highlighted.
+
+```python
+import random
+
+def magic_8_ball():
+    responses = [
+        "As I see it, yes.",
+        "Ask again later.",
+        "Better not tell you now.",
+        "Cannot predict now.",
+        "Don’t count on it.",
+        "My reply is no.",
+        "My sources say no.",
+        "Outlook not so good.",
+        "Outlook good.",
+        "Reply hazy, try again.",
+        "Yes – definitely.",
+        "You may rely on it.",
+        "Absolutely not!",
+        "Go for it!",
+        "No way, José!",
+        "Oh, hell no!",
+        "Hell yes!",
+        "Yes, if you believe hard enough.",
+        "No, but that shouldn't stop you.",
+        "Why would you even ask that?"
+    ]
+
+    while True:
+        question = input("Ask the Magic 8 Ball a question (type 'exit' to quit): ")
+        if question.lower() == 'exit':
+            print("Magic 8 Ball has left the chat.")
+            break
+        else:
+            print(random.choice(responses))
+
+if __name__ == "__main__":
+    magic_8_ball()
+
+```
+
+# Custom Markdoc Tags
+We've extended markdown with custom tags that let us make the docs richer and more interactive.
+
+## Admonitions
```

Comment:
Could we take this opportunity to update some of the styling on these? Specifically, aligning the icon and copy to the top - this has always driven me nuts.

Lmk if that should be considered scope creep for this PR.

Reply:
I can definitely get that put together. If you have any other styling tweaks you're looking for, we can break them out into a separate story. Otherwise, I'll try to get something together for this tomorrow morning.